launchable/test_runners/pytest.py

```diff
@@ -130,6 +136,16 @@ def _pytest_formatter(test_path):
 @launchable.record.tests
 def record_tests(client, json_report, source_roots):
 
+    def data_builder(case: TestCase):
+        metadata = MetadataTestCase.fromelem(case)
+        if metadata and metadata.line is not None:
+            return {
+                # Please note that line numbers start from 0.
```

Comment:
> \# Please note that line numbers start from 0.

Is it common for other test reporters?

Reply:
As far as I researched, it's not common.

Reply:
> it's not common.

If so, should we collect the actual data and adjust at the Launchable side use by combination of this line data and cli profile data, shouldn we?

Reply:
Does "Launchable side" referred to here mean intake side?

Reply:
Launchable side means intake (API) or frontend side

Reply:
Okay.

Reply:
Let me fix it.

Reply:
Ops, wait

Reply:
Sorry, I misunderstood. 
This is the "pytest" runner. So, it's ok to customize for the pytest.

Reply:
Right. No problem.

Reply:
I found out why I misunderstood. This PR title is "Support a line number in file profile"  😄

Reply:
Oh, sorry about that.

Reply:
I've updated it.